packages/server/modules/gatekeeper/graph/resolvers/index.ts

```diff
@@ -124,6 +127,22 @@ export = FF_GATEKEEPER_MODULE_ENABLED
           })
         }
       },
+      ServerWorkspacesInfo: {
+        planPrices: async () => {
+          const getWorkspacePlanPrices = getWorkspacePlanProductPricesFactory({
+            getRecurringPrices: getRecurringPricesFactory({
+              stripe: getStripeClient()
+            }),
+            getWorkspacePlanProductAndPriceIds
+          })
+          const prices = await getWorkspacePlanPrices.fresh()
```

Comment:
oops, left this on .fresh() (skips cache) for testing, i'll switch it back to the cached version when the PR is ready to go